zeppelin-interpreter/pom.xml

```diff
@@ -231,12 +231,6 @@
         <filtering>true</filtering>
       </resource>
     </resources>
-    <plugins>
-      <plugin>
-        <groupId>org.apache.maven.plugins</groupId>
-        <artifactId>maven-dependency-plugin</artifactId>
```

Comment:
It is not necessary to load all libraries into the destination folder, because we only use the shaded interpreter jar.